### PR TITLE
Implement Column Block

### PR DESCRIPTION
**Pull request by mtilburgadobe**
First stab at styling for cards on https://www.eder-gmbh.de/unternehmen/geschaeftsbereiche

Fix [#11](https://github.com/hlxsites/eder-group/issues/11) 

Test URLs:
- Before: https://main--eder-group--hlxsites.hlx.page/unternehmen/geschaeftsbereiche
- After: https://issue-11--eder-group--hlxsites.hlx.page/unternehmen/geschaeftsbereiche

### CODE REVIEW

**Comment by mhaack** [comment]
Some minor UI bugs ...

Screenshot 2023-01-12 at 10 06 52
Screenshot 2023-01-12 at 10 07 01

**Comment by mtilburgadobe** [comment]
> Some minor UI bugs ...
> 
> White space: Screenshot 2023-01-12 at 10 06 52
> 
> Grey column too small: Screenshot 2023-01-12 at 10 07 01

Thanks @mhaack , I fixed the first one, we need to discuss the rest of the columns... it looks like there is a different style being used for the ones at the bottom of the page, if you look on https://www.eder-gmbh.de/unternehmen/geschaeftsbereiche, you can see that only the first two columns use this styling, then it's something different (which actually looks broken on the live site, to be honest)

File: styles/styles.css [hunk 1]
```diff
@@ -229,6 +229,44 @@ a.button.secondary:hover, a.button.secondary:focus, button.secondary:hover, butt
     color: #333;
 }
 
+a.button.coa-button {
```
Comment (mhaack) [inline styles/styles.css:232]: Please don't use `coa-button` we already have primary, secondary buttons. This these once look like secondary with adjusted colors. 
See https://github.com/hlxsites/eder-group/pull/63/files#diff-4e52f2e9b708362ea58d15cca7c25f305187c94f815ba7f8531dea9ebe67c114R31 for example.

**Review by mhaack — APPROVED** [review]
Did some minor style adjustments to use variables for colors.

**Pull request merged**